eng/Versions.props

```diff
@@ -162,36 +160,7 @@
     -->
     <MicrosoftNETCoreAppRuntimeVersion>$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</MicrosoftNETCoreAppRuntimeVersion>
   </PropertyGroup>
-  <!--
-    We ship ref/ assemblies for runtime packages in our targeting targeting pack. When servicing that targeting pack,
-    these assemblies must not change. Must also compile our assemblies against the initial ref/ assemblies for runtime
-    packages. But, need to test against the latest implementation assemblies and ship them in our shared framework.
-    Upshot is we need Major.Minor.0 runtime packages for compilation and the targeting pack and Major.Minor.Latest
-    runtime packages for everything else. This is not an issue for assemblies available in Microsoft.NETCore.App.Ref or
-    Microsoft.Extensions.Internal.Transport because it is next to impossible we would service those packages.
-
-    System.Security.AccessControl should only be referenced in Dependencies.props and RepoTasks.csproj. Because
```

Comment:
What changed to make this no longer necessary?

Reply:
As I mentioned in the description, "dotnet/runtime now keeps `$(AssemblyVersion)` values consistent", specifically for the ref/ assemblies packaged outside Microsoft.NETCore.App.Ref and Microsoft.Extensions.Internal.Transport (where versions were already pinned). Their change is checked in for RC2 and so we no longer need any of these overrides.

Did I miss anything @ericstj @Anipik❔

Reply:
But from the comment we used to include it in our ref pack, even though the Core ref didn't.  Don't we still need to do that?

Reply:
We'll grab the ref/ assemblies from the current (latest) packages rather than the `.0` packages. System.Security.AccessControl in particular comes in due to transitive references; we don't need to reference it directly because we don't need to pin the ref/ assembly.

Reply:
Got it, thanks.